packages/wrangler/src/kv/helpers.ts

```diff
@@ -418,6 +418,13 @@ export function getKVNamespaceId(
 	return namespaceId;
 }
 
+/**
+ * KV namespace ID's must match the pattern `[0-9a-f]{32}`
+ */
+export function isValidKVNamespaceID(id: string) {
+	return /^[0-9a-f]{32}$/.test(id);
```

Comment:
Namespace ids are not case-sensitive.

```suggestion
	return /^[0-9a-f]{32}$/i.test(id);
```

Can you add an appropriate test?